widget/src/text_editor.rs

```diff
@@ -789,14 +789,22 @@ impl Update {
     }
 }
 
-fn motion(key: key::Named) -> Option<Motion> {
+fn motion(key: key::Named, modifiers: keyboard::Modifiers) -> Option<Motion> {
     match key {
+        key::Named::Home | key::Named::ArrowLeft
+            if platform::is_macos_command_pressed(modifiers) =>
```

Comment:
I believe the `if` guard here will affect the `Home` keybinding too.

Reply:
Ah, good catch, I misunderstood how the conditional would be applied and I didn't test on another platform. I've adjusted the match arms accordingly.